doc/man3/OSSL_trace_set_channel.pod

```diff
@@ -180,9 +184,29 @@ point during evaluation.
 
 Traces BIGNUM context operations.
 
-=item B<OSSL_TRACE_CATEGORY_CONF>
+=item B<OSSL_TRACE_CATEGORY_CMP>
 
-Traces details about the provider and engine configuration.
+Traces CMP client and server activity.
+
+=item B<OSSL_TRACE_CATEGORY_STORE>
+
+Traces STORE operations.
+
+=item B<OSSL_TRACE_CATEGORY_DECODER>
+
+Traces decoder operations.
+
+=item B<OSSL_TRACE_CATEGORY_ENCODER>
+
+Traces encoder operations.
+
+=item B<OSSL_TRACE_CATEGORY_ENGINE_REF_COUNT>
+
+Traces decrementing certain ASN.1 structure references.
+
+=item B<OSSL_TRACE_CATEGORY_HTTP>
+
+HTTP client diagnostics.
```

Comment:
*Traces the* HTTP client diagnostics.

Reply:
Changed to: `Traces the HTTP client, such as message headers being sent and received.`